src/Mod/Fem/femresult/resulttools.py

```diff
@@ -411,11 +411,43 @@ def add_principal_stress_std(res_obj):
     res_obj.PrincipalMin = prinstress3
     res_obj.MaxShear = shearstress
     FreeCAD.Console.PrintLog("Added standard principal stresses and max shear values.\n")
+
+    # add critical strain ratio
+    MatMechNon = FreeCAD.ActiveDocument.getObject('MaterialMechanicalNonlinear')
+    if MatMechNon:
+        stress_strain = MatMechNon.YieldPoints
+        if stress_strain:
+            i = -1
+            while stress_strain[i] == "": i -= 1
+            eps_cr_uni = float(stress_strain[i].split(",")[1])
+            alpha = 1.65 * eps_cr_uni
```

Comment:
It is a kind of general rule to add comments for "magic" numbers. For example here I don't get what the 1.65 does.
Can you therefore please add comments.

Also the variable eps_cr_uni might get another name that describes what the variable stores.